docs/sources/operations/request-validation-rate-limits.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Request Validation & Rate-Limit Errors
+weight: 30
+---
+
+# Request Validation & Rate-Limit Errors
+
+Loki will reject requests if they exceed a usage threshold (rate-limit error) or if they are invalid (validation error).
+
+All occurrences of these errors can be observed using the `loki_discarded_samples_total` and `loki_discarded_bytes_total` metrics. The sections below describe the various possible reasons specified in the `reason` label of these metrics.
+
+It is recommended that Loki operators set up alerts or dashboards with these metrics to detect when rate-limits or validation errors occur. 
+
+
+### Terminology
+
+- **sample**: a log line
+- **series**/**stream**: samples with a unique combination of labels
```

Comment:
Fair. I guess it just drives me berserk that we have "stream/series" because as a user I ask myself "why two terms for the same thing?"

I think its not a big deal in the grand scheme of things. Would you be up to just say 'stream'? Then lower down where we have limits that reference 'series' we just say that its a legacy term that means the same thing?

Reply:
Sold! :smiling_face_with_tear: thanks Jen